### PR TITLE
Expose currency on account resource

### PR DESCRIPTION
**Pull request by ct-clearhaus**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -1239,6 +1239,12 @@ GET https://gateway.clearhaus.com/account
   <dd>
     ISO 3166-1 2-letter country code for merchant company.
   </dd>
+  <dt>country
```
Comment (dvaeversted) [inline source/index.html.md:1242]: Shouldn't this be "currency" instead ?

**Comment by tjconcept** [comment]
I'm curious what use case motivated this?

File: source/index.html.md (hunk 1 above)
Comment (ct-clearhaus) [reply source/index.html.md:1242]: Certainly!
```suggestion
  <dt>currency
```

**Comment by ct-clearhaus** [comment]
> I'm curious what use case motivated this?

One merchant can have multiple accounts where the only difference is the funding currency. This allows to easily distinguish which of such accounts a given API key is for. Could be achieved via naming or alike, but `currency` is consistent and always correct; a naming convention is more relaxed and can be mixed up.

**Review by kse-clearhaus — APPROVED** [review]

**Pull request merged**